config/initializers/secret_token.rb

```diff
@@ -4,4 +4,8 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-Crantastic::Application.config.secret_token = '52b1567b80b6c472cd1348eb23da1512c9adae4ce374e5fb48965983cf7708b2e2e972d9c347df91a44bdda6a22b4f79ca9b8e63c9c27d7a02a3b61037a5f7f4'
+Crantastic::Application.config.secret_token = if Rails.env.development? or Rails.env.test?
```

Comment:
Also, `or` and `and` are not the same as `||` and `&&`. The English words are for control flow, not boolean logic.